pkg/kubelet/metrics/metrics.go

```diff
@@ -379,6 +392,64 @@ var (
 		},
 		[]string{"container_state"},
 	)
+	// StartedPodsTotal is a counter that tracks pod sandbox creation operations
```

Comment:
/cc @lilic 

these metrics are currently provided by KSM but I think it would also be useful to be able to compare them to the kubelet-level view